doc/source/FAQ.rst

```diff
@@ -53,7 +53,7 @@ You can use the :meth:`get_neighbors()
 <surprise.prediction_algorithms.algo_base.AlgoBase.get_neighbors>` methods of
 the algorithm object. This is only relevant for algorithms that use a
 similarity measure, such as the :ref:`k-NN algorithms
-<pred_package_knn_inpired>`.
+<pred_package_knn_inspired>`.
```

Comment:
This type also exists in other places so we should either leave it as-is, or fix everywhere:


```py
doc/source/FAQ.rst:56:<pred_package_knn_inpired>`.
doc/source/knn_inspired.rst:1:.. _pred_package_knn_inpired:
surprise/prediction_algorithms/algo_base.py:216:        such as the :ref:`k-NN algorithms <pred_package_knn_inpired>`.
surprise/prediction_algorithms/algo_base.py:264:        such as the :ref:`k-NN algorithms <pred_package_knn_inpired>`.
```

Reply:
Thanks have restored